src/Tribe/Admin_List.php

```diff
@@ -155,6 +158,41 @@ public static function sort_by_event_date( Array $clauses, WP_Query $wp_query )
 			return $clauses;
 		}
 
+		/**
+		 * Defines custom logic for filtering events table by aggregator record.
+		 *
+		 * @param array    $clauses    SQL clauses for fetching posts.
+		 * @param WP_Query $wp_query   A paginated query for items.
+		 *
+		 * @return array                  Modified SQL clauses.
+		 */
+		public static function filter_by_aggregator_record( array $clauses, WP_Query $wp_query ) {
+			// Check for event post type.
+			if ( $wp_query->get( 'post_type' ) !== Tribe__Events__Main::POSTTYPE ) {
+				return $clauses;
+			}
+
+			// Check if filtering by aggregator record.
+			$record_num = intval( tribe_get_request_var( 'aggregator_record', 0 ) );
```

Comment:
```suggestion
			$record_num = (int)tribe_get_request_var( 'aggregator_record', 0 );
```

When possible, so pretty much anywhere, avoid using the `...val()` functions to favor the lighter cast to type.